### PR TITLE
Add support for adding customers to a manual segment

### PR DESCRIPTION
**Pull request by thejanasatan**
This change adds the ability to add customers to a manual segment in CIO.

Will need documentation.

### CODE REVIEW

File: src/Customerio/Request.php [hunk 1]
```diff
@@ -161,6 +161,34 @@ public function anonymousEvent($name, $data)
...
+
+        try {
+            $response = $this->client->post("/api/v1/segments/$segment/add_customers", [
+                'auth' => $this->auth,
+                'json' => $body,
+            ]);
+        } catch (BadResponseException $e) {
+            $response = $e->getResponse();
+        } catch (RequestException $e) {
+            return new Response($e->getCode(), $e->getMessage());
```
Comment (navitronic) [inline src/Customerio/Request.php]: Would assigning the new Response object to `$response` be more consistent here?

File: src/Customerio/Request.php [hunk 2]
```diff
@@ -161,6 +161,34 @@ public function anonymousEvent($name, $data)
         return new Response($response->getStatusCode(), $response->getReasonPhrase());
     }
 
+    public function addToSegment($segment, $users = [])
```
Comment (navitronic) [inline src/Customerio/Request.php]: This class uses phpDoc, to be consistent, you should use it for this method too.

Comment (thejanasatan) [reply src/Customerio/Request.php]: All done :+1:

**Comment by navitronic** [comment]
It also might be worth adding tests for failure responses too, to ensure the behaviour of the try/catch block you’ve added.

**Pull request closed without merging**